src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,42 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { Post } from '../../types/Post';
+import { User } from '../../types/User';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+  users: User[];
+  comments: Comment[];
+};
+
+export const PostInfo: React.FC<Props> = ({ post, users, comments }) => {
+  const user: User | undefined
+  = users.find(person => person.id === post.userId);
```

Comment:
typescript already know about it
But it's up to you
```suggestion
  const user = users.find(person => person.id === post.userId);
```